editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -685,9 +685,8 @@ function useSelectOrLiveModeSelectAndHover(
 
         if (foundTarget != null && foundTargetIsSelected && doubleClick) {
           // for components without passed children doubleclicking enters focus mode
```

Comment:
you might want to revisit the validity of this comment :D

Reply:
Ha! Done